lib/common.sh

```diff
@@ -268,23 +269,23 @@ if [[ "${CAPM3RELEASEBRANCH}" = "release-1.3" ]]; then
   export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.3"}
   export BARE_METAL_OPERATOR_TAG="v0.2.0"
   export KEEPALIVED_TAG="v0.2.0"
-  export BMOBRANCH="${BMOBRANCH:-v0.2.0}"
+  export BMOBRANCH="${BMORELEASEBRANCH:-v0.2.0}"
```

Comment:
This will never take the `v0.2.0` default since we set `BMORELEASEBRANCH` already above.

Reply:
Removed the previous default